packages/create-bison-app/template/package.json.ejs

```diff
@@ -12,8 +12,8 @@
     "build:nexus": "NODE_ENV=development ts-node -P tsconfig.cjs.json --transpile-only graphql/schema.ts",
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
-    "db:migrate": "yarn -s prisma migrate up --experimental && yarn build:prisma",
-    "db:rollback": "yarn prisma migrate down --experimental && yarn build:prisma",
+    "db:migrate": "yarn -s prisma migrate deploy && yarn build:prisma",
+    "db:rollback": "yarn prisma migrate reset && yarn build:prisma",
```

Comment:
That is great feedback! I think we should rename it. It does actually do a full reset and replays migrations. I need to dig back through but there wasn't a rollback per say, I think you literally need to reverse the SQL migration or do via a migration.

Reply:
@cmejet 

Weird... that seems like an oversight on their part. 
--
Actually just realized the new SQL migrations aren't in an `up`/`down` flow. 
Maybe they add that in the next run... 🙏🏼 

-- 

I would def rename this to `reset` if that is the case. 
We can figure out what a true `rollback` looks like later. 👍🏼

Reply:
A good article I found as well about this specifically. I also like that they made a change that when you call reset it will run the seed file if it detects it there. 

https://www.prisma.io/docs/concepts/components/prisma-migrate/prisma-migrate-limitations-issues#lack-of-rollbacks--down-migrations

Reply:
After thinking about this more and chatting with Cory on Nest/CS work -- I get the reasoning... Still feels odd, but I get it.